website/content/docs/release-notes/v0_15_0.mdx

```diff
@@ -231,6 +252,24 @@ description: |-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.15.0
+    <br /><br />
+    (Fixed in 0.15.3)
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Boundary Desktop does not update
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    If you used Homebrew to install Boundary Desktop on macOS, the Desktop client displays a prompt to update automatically, but it never actually updates.
```

Comment:
Thank you for the doc updates! One small thing that i think could be removed "used Homebrew" since the issue is present no matter the installation method on macos.